api/aws_lambdas/utils.py

```diff
@@ -79,6 +76,9 @@ def load_secrets():
 from django_ratelimit.exceptions import Ratelimited  # noqa: E402
 from ninja_extra.exceptions import APIException
 from ninja_jwt.exceptions import InvalidToken  # noqa: E402
+from structlog.contextvars import bind_contextvars  # noqa: E402
+
```

Comment:
Just double checking this is intentional? I know we've ran into errors in the past